### PR TITLE
apps/examples : Add auto generated Kconfig files to gitignore

### PR DESCRIPTION
**Pull request by abhishek-samsung**
After PR #5526, the autogenerated Kconfig and Kconfig_ENTRY files
were being shown in the untracked files. Added them to gitignore.

Signed-off-by: Abhishek Akkabathula <a.akkabathul@samsung.com>

### CODE REVIEW

**Comment by abhishek-samsung** [comment]
@amogh-samsung , please review the PR as the changes are related to #5526

**Review by sunghan-chang — APPROVED** [review]

**Review by jeongchanKim — APPROVED** [review]
Looks good to me.

File: apps/examples/board_specific_demo/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/Kconfig*
+
```
Comment (amogh-samsung) [inline apps/examples/board_specific_demo/.gitignore:2]: I agree, missed out in #5526

File: apps/examples/wifi_manager/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/Kconfig*
+
```
Comment (amogh-samsung) [inline apps/examples/wifi_manager/.gitignore:2]: Looks all good

**Review by amogh-samsung — APPROVED** [review]

**Pull request merged**